Automatic commit: Move 'abap-dev-create-abapdoc' from QA to Production

julieplummer20 opened the pull request.

thecodester commented:
@julieplummer20 I can merge, but I see one change was to say someone can do a specific mission -- you change from group -- but the link continues to point to a group. Why did you change? I want to make sure all the changes were what you wanted.

julieplummer20 commented:
Hi Daniel,
When I looked again, I noticed that it is a mission, but the link still points to the group. (Perhaps it was created as a group at the start?).

So I have updated the link to the MISSION and fixed the link.
And pushed to QA.

Thanks for the catch,
Julie.

From: Daniel Wroblewski <notifications@github.com>
Sent: Donnerstag, 31. Oktober 2019 06:56
To: SAPDocuments/Tutorials <Tutorials@noreply.github.com>
Cc: Plummer, Julie <julie.plummer@sap.com>; Mention <mention@noreply.github.com>
Subject: Re: [SAPDocuments/Tutorials] Automatic commit: Move 'abap-dev-create-abapdoc' from QA to Production (#4134)


@julieplummer20<https://github.com/julieplummer20> I can merge, but I see one change was to say someone can do a specific mission -- you change from group -- but the link continues to point to a group. Why did you change? I want to make sure all the changes were what you wanted.

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://github.com/SAPDocuments/Tutorials/pull/4134?email_source=notifications&email_token=AFNPXTJCL2RXEZCRVJ4HYJTQRJXQXA5CNFSM4JGXFG7KYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOECWUTGQ#issuecomment-548227482>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AFNPXTN7NURIABNQ3RIVU73QRJXQXANCNFSM4JGXFG7A>.
